riff-raff/app/restrictions/RestrictionConfig.scala

```diff
@@ -0,0 +1,32 @@
+package restrictions
+
+import java.util.UUID
+
+import org.joda.time.DateTime
+
+/**
+  * A configuration that restricts which deploys are allowed
+  * @param id Unique ID for CRUD storage and retrival
+  * @param projectName The project name that this applies to
+  * @param stage The stage that this applies to (this can be a regex, but if no special regex characters are found then
```

Comment:
Regex doesn't support Scala like equality (i.e. "a".r != "a".r) so I tend not to use it as a field in case classes  - it breaks the case class semantics.